core/node/groups.go

```diff
@@ -272,17 +234,19 @@ func Online(bcfg *BuildCfg, cfg *config.Config) fx.Option {
 		fx.Provide(p2p.New),
 
 		LibP2P(bcfg, cfg),
-		Providers(cfg),
+		OnlineProviders(cfg),
 	)
 }
 
 // Offline groups offline alternatives to Online units
-var Offline = fx.Options(
-	fx.Provide(offline.Exchange),
-	fx.Provide(Namesys(0)),
-	fx.Provide(offroute.NewOfflineRouter),
-	fx.Provide(provider.NewOfflineProvider),
-)
+func Offline(cfg *config.Config) fx.Option {
```

Comment:
@magik6k I needed to change `Offline` to be a function like `Online`. Let me know if you have concerns about this.